modules/admin/admin-ui/src/main/resources/web/admin/common/styles/api/form/form-option-set.less

```diff
@@ -221,10 +225,30 @@
           background-color: @admin-white;
 
           &:checked {
-            border-color: @admin-option-set-header-bg;
-            box-shadow: 0 0 0 3px @admin-white inset;
-            background-color: @admin-option-set-input-checked;
-            -moz-appearance: none;
+              opacity: 0;
+
+            + label:before {
+              content: "";
+              display: block;
+              position: absolute;
+              top: 2px;
+              left: -15px;
+              height: 14px;
+              width: 14px;
+              background: white;
+              border: 1px solid lightgray;
+              box-shadow: inset 0px 0px 0px 2px white;
```

Comment:
Lets replace border-radius styling with a call to mixin class .box-shadow() from mixins.less

Same for border-radius